spec/openapi.yaml

```diff
@@ -148,6 +148,36 @@ paths:
         '500':
           description: An unexpected error occured.
 
+  /postedtime:
+    get:
+      operationId: fetch-posted-time
+      summary: Returns posted time for the team associated with the provided api key.
+      description: This is a long polling call. The connection will be held open until there is a new posted time event. This is an alternative for the webhook mechanism.
```

Comment:
@tom-haines I think that we should keep the connection open for up to 1 minute, or until time is posted to the team, whichever happens earlier, to avoid hung connectors keeping hold of connections. If the connector is still interested in waiting, they make another request.

Do you agree?

Reply:
Yes, 60 secs is good.
The official timeout is longer, but some implementations consider 72 seconds to be the magic timeout period (but most much longer ~12-30 mins) - so 60 is a safe default.